homework/keyvan_amini_sqldb_assignment.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Wed Oct  2 20:13:24 2019
+
+@author: keyvanamini
+"""
+
+# if not already available, pip install dataset
+
+import pandas as pd
+import numpy as np
+import dataset
+import datetime
+
+#Load data
+employees = pd.read_csv('warehousing-operational-databases/homework/extracts/employees.csv')
```

Comment:
This path looks wrong, given that we are INSIDE the folder warehousing-operational-databases, given the structure of this git repo, no?